docs/instruction/03-runtime.md

```diff
@@ -1,215 +1,270 @@
+Clone the code from GitHub:
+
+```shell
+git clone https://github.com/apache/eventmesh.git
+cd eventmesh/
+```
 
-Download and extract the executable binaries of the latest release from [EventMesh download](https://eventmesh.apache.org/download).For example, with the current latest version, you will get `apache-eventmesh-1.9.0.tar.gz`.
+You can also download the Source Code release from [EventMesh Download](https://eventmesh.apache.org/download) and extract it:
 
-```console
-tar -xvzf apache-eventmesh-1.9.0-bin.tar.gz
-cd apache-eventmesh-1.9.0
+```shell
+wget https://dlcdn.apache.org/eventmesh/1.10.0/apache-eventmesh-1.10.0-source.tar.gz
+tar -xvzf apache-eventmesh-1.10.0-source.tar.gz
+cd apache-eventmesh-1.10.0-src/
 ```
 
-### 2.3 Deploy
+### 3.3 Project Structure Explanation
 
-Edit the `eventmesh.properties` to change the configuration (e.g. TCP port, client blacklist) of EventMesh Runtime. The executable binaries contain all plugins in the bundle, thus there's no need to build them from source code.
+| Module Name                | Description                          |
+| -------------------------- | ------------------------------------ |
+| eventmesh-common           | EventMesh common classes and methods module |
+| eventmesh-connector-api    | EventMesh Connector plugin interface definition module |
+| eventmesh-connector-plugin | EventMesh Connector plugin module    |
+| eventmesh-runtime          | EventMesh Runtime runtime module      |
+| eventmesh-sdk-java         | EventMesh Java client SDK             |
+| eventmesh-starter          | EventMesh local startup project entry point |
+| eventmesh-spi              | EventMesh SPI loading module          |
 
-```console
-vim conf/eventmesh.properties
-```
+> Plugin modules follow the SPI specification defined by EventMesh, and custom SPI interfaces need to be annotated with `@EventMeshSPI`.
+>
+> Plugin instances need to be configured in the corresponding module under the `/main/resources/META-INF/eventmesh` directory with a mapping file for interface and implementation classes. The file name is the fully qualified class name of the SPI interface.
+>
+> The content of the file is the mapping from the plugin instance name to the plugin instance. For details, refer to the `eventmesh-connector-rocketmq` plugin module.
 
-Execute the `start.sh` script to start the EventMesh Runtime server.
+### 3.4 Plugin Explanation
 
-```console
-bash bin/start.sh
+#### 3.4.1 Install Plugins
+
+EventMesh has an SPI mechanism that allows EventMesh to discover and load plugins. There are two ways to install plugins:
+
+- Classpath loading: During local development, you can add dependencies in the `build.gradle` of the `eventmesh-starter` module. For example, to add the Kafka Connector plugin:
+
+```gradle
+dependencies {
+   implementation project(":eventmesh-runtime")
+    // Example: Add the Kafka Connector plugin
+   implementation project(":eventmesh-connectors:eventmesh-connector-kafka")
```

Comment:
eventmesh-connectors do not run in the plugin way described above.

Reply:
Fixed in https://github.com/apache/eventmesh-site/pull/170/commits/dad1f5807f4146a7b06f8eb45e8cb350f5e7f6ae.